Monolithic-Ecommerce/README.md

```diff
@@ -0,0 +1,126 @@
+## Monolithic-Ecommerce App
```

Comment:
README.md has a specific structure that each example needs to implement. See here: https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Reply:
Maybe we could use monolithic-architecture as the folder/module name?

Reply:
I'll revise them as soon as possible